apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/Logger.kt

```diff
@@ -1,22 +0,0 @@
-package com.apollographql.apollo3.api
-
-/**
- * Logger to use for logging by the [com.apollographql.apollo3.ApolloClient]
- */
-interface Logger {
-  /**
-   * Logs the message to the appropriate channel (file, console, etc)
-   *
-   * @param priority the priority to set
-   * @param message message to log
-   * @param t Optional throwable to log
-   * @param args extra arguments to pass to the logged message.
-   */
-  fun log(priority: Int, message: String, t: Throwable?, vararg args: Any)
-
-  companion object {
-    const val DEBUG = 3
-    const val WARN = 5
-    const val ERROR = 6
-  }
-}
```

Comment:
We don't use `Logger` any more at this point. I think the only place where it was used was to debug cache misses which is now exposed through `CacheMissException`